code/modules/mob/living/carbon/xenomorph/mutators/strains/drone/healer.dm

```diff
@@ -148,6 +151,9 @@
 	var/salve_applied_recently = FALSE
 	var/mutable_appearance/salve_applied_icon
 
+	var/trasnferred_amount = 0
```

Comment:
transferred ?

Reply:
Oh my god how did this happen, I'll fix the vars after I get some sleep